instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/camel.yaml

```
@@ -0,0 +1,262 @@
+---
+rules:
+  - bean: org.apache.camel:context=*,type=context,name=*
+    prefix: camel.context.
+    metricAttribute:
+      context: param(context)
+      camelVersion: beanattr(CamelVersion)
+    mapping:
+      ExchangesCompleted:
+        metric: exchange.completed
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges processed successfully since context start-up or the last reset operation.
+      ExchangesFailed:
+        metric: exchange.failed
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges that failed to process since context start-up or the last reset operation.
+      ExchangesInflight:
+        metric: exchange.inflight
+        type: updowncounter
+        unit: "{exchanges}"
+        desc: Indicates the number of exchanges currently transiting the context.
+      ExchangesTotal:
+        metric: exchange.total
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges, passed or failed, processed context start-up or the last reset operation.
+      FailuresHandled:
+        metric: failure.handled
+        unit: "{exchanges}"
+        type: counter
+        desc: Indicates the number of exchanges failed and handled by an ExceptionHandler in the context.
+      ExternalRedeliveries:
+        metric: external.redelivery.count
```

(seems more consistent with above names?)
```suggestion
        metric: external.redelivered
```

---

I prefer the "redelivery.count", it is self-explanatory.

---

wdyt of `external.handled.count` / etc then?

---

wdym? if you are suggesting to change `failure.handled` metric name to `failure.handled.count` for consistency, yeah I like it. Otherwise I don't understand what do you mean with `external.handled.count`